SPECS/boringssl/boringssl.spec

```diff
@@ -107,9 +107,9 @@ find %{buildroot}%{src_install_dir} -type f \( -name "*.a" -o -name "*.lib" -o -
 # Fix non-executable-script warning.
 find %{buildroot}%{src_install_dir} -type f -name "*.sh" -exec chmod +x "{}" +
 # Fix env-script-interpreter error.
-find %{buildroot}%{src_install_dir} -type f -name "*.pl" -exec sed -i 's|#!.*/usr/bin/env perl|#!/usr/bin/perl|' "{}" +
-find %{buildroot}%{src_install_dir} -type f -name "*.py" -exec sed -i 's|#!.*/usr/bin/env python.*|#!/usr/bin/python3|' "{}" +
-find %{buildroot}%{src_install_dir} -type f -name "*.sh" -exec sed -i 's|#!.*/usr/bin/env bash|#!/bin/bash|' "{}" +
+find %{buildroot}%{src_install_dir} -type f -name "*.pl" -exec sed -i 's|#!.*%{_bindir}/env perl|#!/usr/bin/perl|' "{}" +
```

Comment:
We need to be careful with spec linting in such cases (it's not perfect). The `/usr/bin` path replaced with `%{_bindir}` is part of a script from the package's source. It may not always follow what Mariner's definition of the `_bindir` macro is. For instance, if we ever modified that macro, this would break, because `sed` would no longer find for the right path inside this script. We could (and should), however, use that macro for the path replacement. So for this line:

```bash
find %{buildroot}%{src_install_dir} -type f -name "*.pl" -exec sed -i 's|#!.*/usr/bin/env perl|#!%{_bindir}/perl|' "{}" +
```

Same idea goes for the other two lines. I would only first check where Mariner installs `perl`, `python3`, and `bash` to make sure we're using the correct paths. For instance, I'm not sure if `bash` is under `/bin/bash` or maybe elsewhere. I think we also don't have a macro for `/bin`.